apps/dav/lib/Command/ImportCalendar.php

```diff
@@ -0,0 +1,189 @@
+		}
+		if (!$calendar->isWritable()) {
+			throw new InvalidArgumentException("Calendar <$calendarId> is not writeable");
+		}
+		if ($calendar->isDeleted()) {
+			throw new InvalidArgumentException("Calendar <$calendarId> is deleted");
+		}
+		// construct options object
+		$options = new CalendarImportOptions();
+		$options->setSupersede($supersede);
+		if ($errors !== null) {
+			$options->setErrors($errors);
+		}
+		if ($validation !== null) {
+			$options->setValidate($validation);
+		}
+		$options->setFormat($format);
+		// evaluate if a valid location was given and is usable otherwise default to stdin
+		$timeStarted = microtime(true);
+		if ($location !== null) {
+			$input = fopen($location, 'r');
+			if ($input === false) {
+				throw new InvalidArgumentException("Location <$location> is not valid. Can not open location for read operation.");
+			}
+			try {
+				$outcome = $this->importService->import($input, $calendar, $options);
+			} finally {
+				fclose($input);
+			}
+		} else {
+			$input = fopen('php://stdin', 'r');
+			if ($input === false) {
+				throw new InvalidArgumentException('Can not open stdin for read operation.');
+			}
+			try {
+				$tempPath = $this->tempManager->getTemporaryFile();
+				$tempFile = fopen($tempPath, 'w+');
+				while (!feof($input)) {
+					fwrite($tempFile, fread($input, 8192));
+				}
+				fseek($tempFile, 0);
+				$outcome = $this->importService->import($tempFile, $calendar, $options);
+			} finally {
+				fclose($input);
+				fclose($tempFile);
+			}
+		}
+		$timeFinished = microtime(true);
+
+		// summarize the outcome
+		$totalCreated = 0;
+		$totalUpdated = 0;
+		$totalSkipped = 0;
+		$totalErrors = 0;
+
+		if ($outcome !== []) {
+			if ($showCreated || $showUpdated || $showSkipped || $showErrors) {
+				$output->writeln('');
+			}
+			foreach ($outcome as $id => $result) {
+				if (isset($result['outcome'])) {
+					switch ($result['outcome']) {
+						case 'created':
+							$totalCreated++;
+							if ($showCreated) {
+								$output->writeln(['created: ' . $id]);
+							}
+							break;
+						case 'updated':
+							$totalUpdated++;
+							if ($showUpdated) {
+								$output->writeln(['updated: ' . $id]);
+							}
+							break;
+						case 'exists':
+							$totalSkipped++;
+							if ($showSkipped) {
+								$output->writeln(['skipped: ' . $id]);
+							}
+							break;
+						case 'error':
+							$totalErrors++;
+							if ($showErrors) {
+								$output->writeln(['errors: ' . $id]);
+								$output->writeln($result['errors']);
+							}
+							break;
+					}
+				}
+			}
+		}
+		$output->writeln([
+			'',
+			'Import Completed',
+			'================',
+			'Execution Time: ' . (string)($timeFinished - $timeStarted) . ' sec',
+			'Total Created: ' . (string)$totalCreated,
+			'Total Updated: ' . (string)$totalUpdated,
+			'Total Skipped: ' . (string)$totalSkipped,
+			'Total Errors: ' . (string)$totalErrors,
```

Comment:
```suggestion
			'Execution Time: ' . ($timeFinished - $timeStarted) . ' sec',
			'Total Created: ' . $totalCreated,
			'Total Updated: ' . $totalUpdated,
			'Total Skipped: ' . $totalSkipped,
			'Total Errors: ' . $totalErrors,
```

Concatenation automatically casts to string